src/postgraphile/http/createPostGraphileHttpRequestHandler.ts

```diff
@@ -110,6 +115,34 @@ const isPostGraphileDevelopmentMode = process.env.POSTGRAPHILE_ENV === 'developm
 const debugGraphql = Debugger('postgraphile:graphql');
 const debugRequest = Debugger('postgraphile:request');
 
+function parseExplainOptions(req: IncomingMessage): ExplainOptions {
+  const explainOptions = {};
+  const header = req.headers['x-postgraphile-explain-options'];
+  if (!header) {
+    return explainOptions;
+  }
+  const headerValues = typeof header === 'string' ? header.split(',') : header;
+  for (const headerValue of headerValues) {
+    const [option, value] = headerValue.split('=').map(s => s.toLowerCase().trim());
+    switch (option) {
+      case 'format':
+        if (['text', 'xml', 'json', 'yaml'].includes(value)) {
+          explainOptions[option] = value;
+        } else {
+          console.warn(`Ignoring invalid value '${value}' for explain option '${option}'.`);
+        }
+        break;
+      default:
+        if (['true', 'false'].includes(value)) {
+          explainOptions[option] = value === 'true';
+        } else {
+          console.warn(`Ignoring invalid value '${value}' for explain option '${option}'.`);
+        }
+    }
+  }
+  return explainOptions;
+}
```

Comment:
Do we really need custom parsing for this? I feel like we should just use `querystring` or JSON or similar.